<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <item row="2" column="0" rowspan="7" colspan="7">
     <layout class="QGridLayout" name="gridLayout_graph"/>
    </item>
    <item row="0" column="2" colspan="5">
     <widget class="QLineEdit" name="text_input_path">
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>Path to input audio file...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="8" rowspan="9">
     <widget class="QPlainTextEdit" name="text_result_paths">
      <property name="plainText">
       <string>Here you will see starts of potential time segments...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_open_file">
      <property name="text">
       <string>Choose Input File</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Input Path:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="7">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="2" column="4">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>sec</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QPushButton" name="pushButton_mf_analysis">
        <property name="text">
         <string>Male/Female Segments</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="pushButton_SVM">
        <property name="text">
         <string>SVM</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="line_edit_start_fragment"/>
      </item>
      <item row="4" column="6">
       <widget class="QPushButton" name="pushButton_VAD">
        <property name="text">
         <string>Voice Activation Detection</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Analysis Starter Buttons</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>sec</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="line_edit_end_fragment"/>
      </item>
      <item row="2" column="6">
       <widget class="QPushButton" name="pushButton_SIlence">
        <property name="text">
         <string>Silence Segments</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>End of Fragment:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Start of the Fragment:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pushButton_play_fragment">
        <property name="text">
         <string>Play Fragment</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QPushButton" name="pushButton_stop_fragment">
        <property name="text">
         <string>Stop Fragment</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="push_button_view_fragement">
        <property name="text">
         <string>View Fragment</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="11" column="0" colspan="9">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
